intake/catalog/local.py

```diff
@@ -503,17 +503,8 @@ def register_plugin_module(mod):
 
 
 def get_dir(path):
-    if "://" in path:
-        protocol, _ = split_protocol(path)
-        out = get_filesystem_class(protocol)._parent(path)
-        if "://" not in out:
-            # some FSs strip this, some do not
-            out = protocol + "://" + out
-        return out
-    path = make_path_posix(os.path.join(os.getcwd(), os.path.dirname(path)))
-    if path[-1] != "/":
-        path += "/"
-    return path
+    """Get the parent directory of the path"""
+    return f"{PurePath(path).parent.as_posix().replace(':/', '://')}/"
```

Comment:
This now fails for Windows paths

Reply:
Sorry, I don't get it. On my Windows laptop, all the tests from `test_local.py` are completed successfully except for `def test_default_expansions()`, but this test was failing before.

If you are worrying about converting the path to POSIX, the `get_filesystem_class(protocol)._parent(path)` does it also by calling `make_path_posix(path)`:
```python
    @classmethod
    def _parent(cls, path):
        path = cls._strip_protocol(path).rstrip("/")
        if "/" in path:
            return path.rsplit("/", 1)[0]
        else:
            return cls.root_marker

    @classmethod
    def _strip_protocol(cls, path):
        path = stringify_path(path)
        if path.startswith("file://"):
            path = path[7:]
        elif path.startswith("file:"):
            path = path[5:]
        return make_path_posix(path).rstrip("/") or cls.root_marker
```